resources/views/ItemList/Components/Filter/ItemFilter.twig

```diff
@@ -61,7 +61,7 @@
             <div class="form-check" v-for="value in facet.values" :key="value.id">
                 <input :id="'option-' + value.id" class="form-check-input hidden-xs-up" type="checkbox" :checked="value.selected" @change="updateFacet(value)" :disabled="isLoading">
                 <label :for="'option-' + value.id" class="form-check-label" v-text="value.name" rel="nofollow"></label>
-                <div class="filter-badge" v-text="value.count"></div>
+                <div v-if="!value.selected" class="filter-badge" v-text="value.count"></div>
```

Comment:
Does this work for all filter types, also including vendor image, or others?

Reply:
Good call, it wasn't working with images. All the others look fine.